test/integration/help.test.ts

```diff
@@ -0,0 +1,13 @@
+import { runCodeServerCommand } from "../utils/runCodeServerCommand"
+
+// NOTE@jsjoeio
+// We have this test to ensure that native modules
+// work as expected. If this is called on the wrong
+// platform, the test will fail.
```

Comment:
🙅🏼‍♂️ I know we generally avoid comments like this and prefer putting this info in commits. But I feel like this should be front and center so that future developers don't accidentally delete this test.

I can be convinced otherwise though.

Reply:
Yeah this seems like a reasonable comment to me because it is not self-apparent why we test `--help`.  

One way around that would be to change the wording of the test, maybe something like `describe("native-modules")` and `test("it should trigger native modules by running --help)` so the reasoning is baked into the code but IMO either way is good (I probably would have gone the comment route as well).

Reply:
I hadn't thought of going with the test name like that! Good point.

Cool! Glad we're in agreement. Happy to change later if we change our minds as well!